src/man/sssd.conf.5.xml

```diff
@@ -1766,14 +1766,21 @@ p11_uri = library-description=OpenSC%20smartcard%20framework;slot-id=2
                             will be filtered out and ssh keys will be generated
                             from all valid certificates.
                         </para>
+                        <para>
+                            If no rules are configured using 'all_rules' will
+                            enable a default rules which enables all
```

Comment:
I'm not native speaker but I think it should be "a default rule" or "default rules"

Reply:
Hi,

thanks for the review, the new version should fix the typo.

bye,
Sumit